content/pt-br/docs/reference/glossary/secret.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Secret
+id: secret
+date: 2021-08-24
+full_link: /docs/concepts/configuration/secret/
+short_description: >
+  Armazena dados sensíveis, como senhas, tokens OAuth e chaves SSH.
+
+aka: 
+tags:
+- core-object
+- security
+---
+ Armazena dados sensíveis, como senhas, tokens OAuth e chaves SSH.
+
+<!--more--> 
+
+Permite mais controle com relação a como as informações sensíveis são armazenadas e reduz o risco de exposição acidental, incluindo [encriptação](/docs/tasks/administer-cluster/encrypt-data/#ensure-all-secrets-are-encrypted) em disco. Um {{< glossary_tooltip text="Pod" term_id="pod" >}} pode referenciar o Secret como um arquivo em um volume montado, ou o kubelet pode usar este dado quando baixa imagens para um pod. Secrets são úteis para dados confidenciais, enquanto [ConfigMaps](/docs/tasks/configure-pod-container/configure-pod-configmap/) são úteis para dados não-confidenciais.
```

Comment:
So, here maybe a "bonus" (also, would recommend to look into english translation and improve it).

There has been a discussion on pass on whether secrets values are or not encrypted (well, base64 is encoding, not encryption).

So what do you think of us taking this opportunity to:

* In this PR, add a note saying something like "Vale notar que os valores dos secrets são CODIFICADOS, mas não CRIPTOGRAFADOS em formato Base64"
* A new PR improving the English docs

Reply:
This is an interesting point. I was working with the assumption that the parts that speak to encryption referred specifically to the possibility of encrypting data at rest (as explained on https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/), but now re-reading I can see some ambiguity here. 😅 

Should we state that, by default, secrets are only codified in base64, but can be encrypted if the cluster administrator/operator takes steps towards doing so?

Reply:
Opened #29985 for changing the English version.